.github/workflows/ci.yml

```diff
@@ -209,14 +196,91 @@ jobs:
           notify_when: 'failure'
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_NOTIFY_BUILD }}
-          RUSTFLAGS: -D warnings
 
-  # TODO: https://github.com/FuelLabs/fuel-indexer/issues/269
-  cargo-test-workspace-all-features:
+  cargo-clippy:
     if: github.event_name != 'release' && github.event.action != 'published'
     needs:
       - cancel-previous-runs
-      - get-workspace-members
+      - cargo-toml-fmt-check
+      - check-rustc
+    runs-on: ubuntu-latest
+    services:
+      postgres:
```

Comment:
I don't think we need postgres for `clippy`?

Reply:
Oh, you know, I included it for `sqlx` but I forgot that we had `SQLX_OFFLINE=1` set at the very beginning of the file. I'll change.